python/phonenumbers/phonenumberutil.py

```diff
@@ -2286,7 +2286,7 @@ def _test_number_length(national_number, metadata, numtype=PhoneNumberType.UNKNO
         return ValidationResult.INVALID_LENGTH
 
     actual_length = len(national_number)
-    # This is safe because there is never an overlap beween the possible lengths and the local-only
+    # This is safe because there is never an overlap between the possible lengths and the local-only
```

Comment:
BTW, this is an [upstream typo](https://github.com/google/libphonenumber/blob/master/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L2564) so you could fix there too if you're feeling energetic&hellip;